### PR TITLE
Testing setup

### PR DESCRIPTION
**Pull request by spanuska**
This PR adds a test framework and passing tests to the app.
In index.js, I've removed unused code, and abstracted the date-related logic to a new module weddingDates.js; I've added tests for both of those.

I removed the npm module 'ordinal', as it seems that it is no longer supported by the author. I've replaced it with another package that provides the same functionality.

Let me know what questions you have about these changes. I'd love to hear your feedback!

### CODE REVIEW

File: index.test.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+var expect = require('chai').expect;
+var countdown = require('./index');
+var moment = require('moment');
+
+describe("countdown message", function() {
+  it("is a string", function() {
```
Comment (johnkpaul) [inline index.test.js]: For so many assertions, and multiple evaluations of countdown.composeMessage, what do you think about splitting this into multiple tests? This is doing much more than checking if the return value is a string.

Comment (spanuska) [inline index.test.js]: Yes, good catch. That's not the correct description for the test. It's really testing that the message is correctly interpolating the variables. I'll revise.

**Comment by johnkpaul** [comment]
Thank you so much for this PR! It looks great and I am very happy to have tests added to the project.

File: index.test.js (hunk 1 above)
Comment (johnkpaul) [inline index.test.js]: 👍

File: index.test.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+var expect = require('chai').expect;
+var countdown = require('./index');
+
+describe("countdown message", function() {
+  it("is a string with correctly interpolated variables", function() {
```
Comment (spanuska) [inline index.test.js:5]: @johnkpaul: how is this?

Comment (johnkpaul) [inline index.test.js:5]: 👍

**Pull request merged**